src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -29,22 +32,46 @@ def lldp_entry_table(if_name):
     return b'LLDP_ENTRY_TABLE:' + if_name
 
 
+def if_entry_table(if_name):
+    """
+    :param if_name: given interface to cast.
+    :return: PORT_TABLE key.
+    """
+    return b'PORT_TABLE:' + if_name
+
+
+def lag_entry_table(lag_name):
+    """
+    :param lag_name: given lag to cast.
+    :return: LAG_TABLE key.
+    """
+    return b'LAG_TABLE:' + lag_name
+
+
 def get_index(if_name):
     """
     OIDs are 1-based, interfaces are 0-based, return the 1-based index
     Ethernet N = N + 1
+    PortChannel N = N + 1000
     """
     return get_index_from_str(if_name.decode())
 
+
 def get_index_from_str(if_name):
     """
     OIDs are 1-based, interfaces are 0-based, return the 1-based index
     Ethernet N = N + 1
+    PortChannel N = N + 1000
     """
-    match = re.match(SONIC_ETHERNET_RE_PATTERN, if_name)
-    if match:
-        n = match.group(1)
-        return int(n) + 1
+    patterns = {
+        SONIC_ETHERNET_RE_PATTERN: 1,
+        SONIC_PORTCHANNEL_RE_PATTERN: 1000
```

Comment:
>1000 [](start = 38, length = 4)

Magic number. Give a name and some comment? May be '1' above is in the same condition.